Resources/Prototypes/_Harmony/Catalog/uplink_catalog.yml

```
@@ -10,6 +10,30 @@
   categories:
   - UplinkExplosives
 # Ammo
+
+# For the Bulldog
+- type: listing
+  id: UplinkShotgunShotgunBuckshotMagazineHarmony
```

Shotgun appears to be repeated twice in the id
```suggestion
  id: UplinkShotgunBuckshotMagazineHarmony
```